app/components/AppUpdateNotification.tsx

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable react/no-unused-prop-types */
+import React, {
+    useState,
+    useCallback,
+    useEffect,
+    useMemo,
+    Dispatch,
+    SetStateAction,
+    ComponentType,
+} from 'react';
+
+import CogIcon from '@resources/svg/settings.svg';
+import ErrorIcon from '@resources/svg/logo-error.svg';
+import CheckmarkIcon from '@resources/svg/logo-checkmark.svg';
+import Button from '~/cross-app-components/Button';
+import { NotificationLevel } from '~/features/NotificationSlice';
+import Loading from '~/cross-app-components/Loading';
+
+import Notification from './Notification';
+import Countdown from './Countdown';
+
+enum UpdateStatus {
+    Available,
+    Downloading,
+    Verifying,
+    Success,
+    Error,
+}
+interface Props {
+    onUpdate(): void;
+    onPostpone(): void;
+}
+
+interface State {
+    status: UpdateStatus;
+    message?: string;
+}
+
+interface ChildProps extends Props, State {
+    setState: Dispatch<SetStateAction<State>>;
+}
+
+const AvailableChild = ({ onUpdate, setState, onPostpone }: ChildProps) => {
+    const update = useCallback(() => {
+        onUpdate();
+        setState({ status: UpdateStatus.Downloading });
+    }, [onUpdate, setState]);
+
+    return (
+        <>
+            <div className="flex alignCenter">
+                <CogIcon width="21" height="22" />
+                <span className="mL5">New updates are available</span>
```

Comment:
Perhaps it could state the version that is available like:
```
Version 1.2.0 is available
```

Reply:
Talked it over with @jens-concordium as well, and we agreed that version number could be useful to know in different scenarios. It is now included.